travis/travis_run.sh

```diff
@@ -47,8 +47,8 @@ mpirun -np 2 python -c "import toast.tests; toast.tests.run()"
 cd examples/
 # Disable tests with PySM until https://github.com/astropy/astropy/pull/9190
```

Comment:
do you see the same error on Travis?

Reply:
Yes, the MPI-enabled examples produce the astropy error on Travis.

Please see the end of the Travis log here:
https://travis-ci.org/hpc4cmb/toast/jobs/580932159